docs/server/upgrading.rst

```diff
@@ -3,170 +3,222 @@
 Upgrading
 =========
 
-These are the instructions for upgrading Pootle from an older version to a new
-release.
+These are the instructions for upgrading Pootle from an older version to the
+current release.
+
+
+.. _upgrading#system-backup:
+
+Backup your system
+------------------
+
 
 .. warning::
 
-  When upgrading please ensure that you:
+   Before upgrading we **strongly** recommend that you
+   :ref:`backup your current system <backup>`.
 
-  - **Carefully read this page** before proceeding
-  - Make all the recommended **backups**
-  - Try to **follow these instructions** as closely as possible
 
+.. _upgrading#db-migration:
 
-This page is divided in three sections:
+Migrate your database
+---------------------
 
-1. Preparatory tasks that should be performed before upgrading.
-2. Detailed steps to actually perform the upgrade.
-3. Suggested tasks to fine tune the setup after upgrading.
+If you are currently using sqlite for your database you will need to 
+:doc:`migrate to either MySQL or PostgreSQL <database_migration>`
+before you upgrade.
 
 
-.. _upgrading#preparatory-tasks:
+.. _upgrading#latest-changes:
 
-Preparatory tasks
------------------
+Latest changes
+--------------
 
-Before upgrading Pootle to a newer version, make sure to go through this
-checklist.
+Before upgrading Pootle its worth checking for
+:doc:`important changes </releases/index>` since the version that you are
+upgrading from.
```

Comment:
Before upgrading Pootle its worth checking for important changes  since the version that you are upgrading from.

Rather....
Before upgrading Pootle familiarise yourself with the important changes that have been implemented since your current Pootle version.